documentation/airport-guides/kdab.md

```diff
@@ -0,0 +1,13 @@
+# KDAB - Daytona Beach International Airport
+
+## About SAT
+Daytona Beach International Airport (IATA: DAB, ICAO: KDAB, FAA LID: DAB) is a county-owned airport located three miles (5 km) southwest of Daytona Beach, next to Daytona International Speedway, in Volusia County, Florida, United States. The airport has 3 runways, a six-gate domestic terminal, and an international terminal. Daytona Beach is the headquarters of Embry-Riddle Aeronautical University. (Wikipedia)
+
+## Airspace
+The DAB airspace covers multiple smaller airports in the region from the surface to 10,000'. It is bordered to the north and west by Jacksonville Center and to the south by the Central Florida TRACON (F11) and Miami Center.
+
+## Procedures
+DAB has one conventional SID and one RNAV STAR. Additionally, the airspace has a significant number of Victor airways and RNAV T-routes.
```

Comment:
It might be worth mentioning the overflights here. Most users will have never seen one in the sim before so a very short explanation on what they need to do with them would be good to have here.